README.md

```diff
@@ -157,38 +166,82 @@ OR
 
 ##### Auto linking
 
-`react-native link react-native-sdk-beta`
+`react-native link onewelcome-react-native-sdk`
 
 ##### Manual installation
 
-
 #### iOS
 
 1. In XCode, in the project navigator, right click `Libraries` ➜ `Add Files to [your project's name]`
-2. Go to `node_modules` ➜ `react-native-sdk-beta` and add `RNOneginiSdk.xcodeproj`
+2. Go to `node_modules` ➜ `onewelcome-react-native-sdk` and add `RNOneginiSdk.xcodeproj`
 3. In XCode, in the project navigator, select your project. Add `libRNOneginiSdk.a` to your project's `Build Phases` ➜ `Link Binary With Libraries`
 4. Run your project (`Cmd+R`)<
 
 #### Android
 
 1. Open up `android/app/src/main/java/[...]/MainActivity.java`
-  - Add `import com.onegini.mobile.RNOneginiSdkPackage;` to the imports at the top of the file
-  - Add `new RNOneginiSdkPackage()` to the list returned by the `getPackages()` method
+
+- Add `import com.onegini.mobile.RNOneginiSdkPackage;` to the imports at the top of the file
+- Add `new RNOneginiSdkPackage()` to the list returned by the `getPackages()` method
```

Comment:
I've been trying this branch out for a while. Since autolinking hasn't been working yet, I needed to prevent autolinking via `react-native.config.js` and do it manually. I know there is "todo" note in `RNOneginiSdkPackage.java` but here are some things I noticed to make it work via manual linking and reading through the documentation.

Shouldn't be `RNOneginiSdkPackage` imported in `MainApplication.java` instead of `MainActivity.java`, since there've been packages linked automatically?
Also `RNOneginiSdkPackage` is in package `com.onegini.mobile.sdk.reactnative` so it should be `import com.onegini.mobile.sdk.reactnative.RNOneginiSdkPackage`.